packages/expo-updates/ios/EXUpdates/SelectionPolicy/EXUpdatesLoaderSelectionPolicyFilterAware.m

```diff
@@ -28,6 +28,30 @@ - (BOOL)shouldLoadNewUpdate:(nullable EXUpdatesUpdate *)newUpdate withLaunchedUp
   return [launchedUpdate.commitTime compare:newUpdate.commitTime] == NSOrderedAscending;
 }
 
+- (NSArray<EXUpdatesUpdate *> *)outdatedUpdatesWithLaunchedUpdate:(EXUpdatesUpdate *)launchedUpdate updates:(NSArray<EXUpdatesUpdate *> *)updates filters:(nullable NSDictionary *)filters
+{
+  if (!launchedUpdate) {
+    return @[];
+  }
+
+  NSMutableArray<EXUpdatesUpdate *> *outdatedUpdates = [NSMutableArray new];
+  for (EXUpdatesUpdate *update in updates) {
+    // ignore any updates whose scopeKey doesn't match that of the launched update
+    if (![launchedUpdate.scopeKey isEqualToString:update.scopeKey]) {
+      continue;
+    }
+    // also ignore any updates that don't match the filters of the launched update
```

Comment:
Does this include checking for the same RTV? Arguably updates that target a different RTV shouldn't be compared with the launched update.